stories/object-attribute/object-attribute.stories.js

```diff
@@ -0,0 +1,33 @@
+export default {
+    title: 'Dev/Object Attribute',
+    components: ['object-attribute']
+};
+
+export const Text = () => `
+
+<div class="fd-object-attribute" id="standardtext">Standard Object Attribute Text</div>
+`;
```

Comment:
Remove first newline from each story to fix empty jest storyshots.
```suggestion
export const Text = () => `<div class="fd-object-attribute" id="standardtext">Standard Object Attribute Text</div>
`;
```

Reply:
@sKudum the first new line needs to be removed too.